services/app-api/src/resolvers/user/updateCMSUser.ts

```diff
@@ -39,56 +43,81 @@ export function updateCMSUserResolver(
             divisionAssignment = undefined
         }
 
+        if (!stateAssignments && !divisionAssignment) {
+            const errMsg =
+                'No state assignments or division assignment provided'
+            logError('updateCmsUser', errMsg)
+            setErrorAttributesOnActiveSpan(errMsg, span)
+            throw new UserInputError(errMsg, {
+                code: 'BAD_USER_INPUT',
+                cause: 'NO_ASSIGNMENTS',
+            })
+        }
+
         // validate division assignment and throw an error if invalid
-        if (divisionAssignment !== undefined) {
+        if (divisionAssignment) {
             if (!isValidCmsDivison(divisionAssignment)) {
                 const errMsg = 'Invalid division assignment'
                 logError('updateCmsUser', errMsg)
                 setErrorAttributesOnActiveSpan(errMsg, span)
                 throw new UserInputError(errMsg, {
```

Comment:
I _think_ that b/c you are throwing a UserInputError here, there is no need to specify code: 'BAD_USER_INPUT', unless adding extensions overrides the code to nothing if not set. 

We probably want to have our own errors library that sets all of this with a function at some point.

Reply:
Correct! You don't need to specify the `code` in when using these error classes from `apollo-server-lambda`. What should be changed is the second argument. I believe the entire second argument is the `extensions` object, so in this case, the error object would contain an extension inside the extension. 

```typescript
throw new UserInputError(errMsg, {
   argumentName: 'divisionAssignment',
   argumentValues: divisionAssignment,
   cause: 'INVALID_DIVISION_ASSIGNMENT'
})
```